scrapers/USA/KY/index.js

```diff
@@ -0,0 +1,51 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as transform from '../../../lib/transform.js';
+import * as geography from '../../../lib/geography.js';
+
+const scraper = {
+  country: 'USA',
+  state: 'KY',
+  priority: 1,
+  url: 'https://datawrapper.dwcdn.net/BbowM/23/',
```

Comment:
Is this Kentucky Health Organization? If so, let's add
```suggestion
  url: 'https://datawrapper.dwcdn.net/BbowM/23/',
  source: {
    name: 'Kentucky Health Organization',
    url: 'http://wherever.this.comes.from'
  },
```